### PR TITLE
Amazon ECS doc update (SUMO-206570)

### PR DESCRIPTION
**Pull request by JV0812**
## Purpose of this pull request

This pull request (PR) ...

Issue number: 

## Select the type of change:

- [ ] Minor Changes - Typos, formatting, slight revisions
- [x] Update Content - Revisions and updating sections
- [ ] New Content - New features, sections, pages, tutorials
- [ ] Site and Tools - Updates, maintenance, and new packages for the site, Gatsby, React, etc

### CODE REVIEW

File: docs/integrations/amazon-aws/elastic-container-service-container-insights-cloudwatch.md [hunk 1]
```diff
@@ -56,48 +56,16 @@ Parse Expression:
...
-13. **Metadata: **Add an **account** field to the source and assign it a value that is a friendly name/alias to your AWS account from which you are collecting metrics. This name will appear in the Sumo Logic Explorer View. Metrics can be queried via the `account field`.
-14. Click **Save**.
+Sumo Logic supports collecting metrics using two source types:
+
+  * Configure an [AWS Kinesis Firehose for Metrics Source](/docs/send-data/hosted-collectors/amazon-aws/aws-kinesis-firehose-metrics-source) (recommended); or
+  * Configure an [Amazon CloudWatch Source for Metrics](/docs/send-data/hosted-collectors/amazon-aws/amazon-cloudwatch-source-metrics)
+  
+**Namespace** for **Amazon Elastic Container Service (ECS) using Container Insights and CloudWatch** service is **AWS/ECS**.
+  
+**Metadata**: Add an **account** field to the source and assign it a value which is a friendly name / alias to your AWS account from which you are collecting metrics. This name will appear in the Sumo Logic Explorer View. Metrics can be queried via the `account field`.
```
Comment (kimsauce) [inline docs/integrations/amazon-aws/elastic-container-service-container-insights-cloudwatch.md:68]:
```suggestion
**Metadata**: Add an **account** field to the source and assign it a value which is a friendly name / alias to your AWS account from which you are collecting metrics. This name will appear in the Sumo Logic [Explore](/docs/dashboards-new/explore-view) view. Metrics can be queried via the `account field`.
```

File: docs/integrations/amazon-aws/elastic-container-service-container-insights-cloudwatch.md [hunk 2]
```diff
@@ -452,14 +443,17 @@ _sourceCategory=ecs* (DeleteCluster or DeleteService or DeregisterContainerInsta
...
 To install the app:
 
 Locate and install the app you need from the **App Catalog**. If you want to see a preview of the dashboards included with the app before installing, click **Preview Dashboards**.
 
 1. From the **App Catalog**, search for and select the app. 
-2. Select the **With Container Insights and Traces** version and click **Add to Library**. Version selection applies only to a few apps currently. For more information, see the [Install the Apps from the Library](/docs/get-started/library).
+2. Select the **With Container Insights and Traces** version and click **Add to Library**. 
+:::note
+Version selection applies only to a few apps currently. For more information, see the [Install the Apps from the Library](/docs/get-started/library).
+:::
```
Comment (kimsauce) [inline docs/integrations/amazon-aws/elastic-container-service-container-insights-cloudwatch.md:456]:
```suggestion
```

Comment (JV0812) [reply docs/integrations/amazon-aws/elastic-container-service-container-insights-cloudwatch.md:456]: Dev guys needed note note. So as per the request from them I have changed the step from a note.

Comment (kimsauce) [reply docs/integrations/amazon-aws/elastic-container-service-container-insights-cloudwatch.md:456]: @JV0812 From what I understand, this note is obsolete. It was written 1-2 years ago Docs-wide, when App versioning was just coming out in Beta. But now feature is ubiquitous.

Comment (JV0812) [reply docs/integrations/amazon-aws/elastic-container-service-container-insights-cloudwatch.md:456]: Okay, got it. Thanks for this info @kimsauce

**Review by kimsauce — APPROVED** [review]

**Pull request merged**